tests/behat/features/global/GDPR.feature

```diff
@@ -9,9 +9,10 @@ Feature: GDPR Links
 #    Given I am on "/"
 #    Then I should be on "/"
 
-  @safe
+  @api @safe
```

Comment:
You've got a lot of tests going on in this package. 👍

Reply:
check out https://github.com/SU-HSDO/suhumsci/pull/97/files
i've got 100% coverage on the courses module now. https://571-129976545-gh.circle-artifacts.com/0/var/www/html/artifacts/coverage-html/index.html

Reply:
oh, nice!